test/parallel_api/ranges/std_ranges_is_sorted_until.pass.cpp

```diff
@@ -0,0 +1,39 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "std_ranges_test.h"
+
+std::int32_t
+main()
+{
+#if _ENABLE_STD_RANGES_TESTING
+    using namespace test_std_ranges;
+    namespace dpl_ranges = oneapi::dpl::ranges;
+
+    auto is_sorted_until_checker = TEST_PREPARE_CALLABLE(std::ranges::is_sorted_until);
+
+    test_range_algo<0>{big_sz}(dpl_ranges::is_sorted_until, is_sorted_until_checker, std::ranges::less{});
```

Comment:
Could you also check the interface without a predicate?

Reply:
added

Reply:
I was not able to find these changes. It seems it was not added.

Reply:
Ok, I will double check similar algo with comparator.

Reply:
added